docs/source/llms/tracing/index.rst

```diff
@@ -807,6 +807,41 @@ To enable async logging for tracing, call :py:func:`mlflow.config.enable_async_l
 
 Note that the async logging does not fully eliminate the performance overhead. Some backend calls still need to be made synchronously and there are other factors such as data serialization. However, async logging can significantly reduce the overall overhead of logging traces, empirically about ~80% for typical workloads.
 
+Using OpenTelemetry Collector for Exporting Traces
+--------------------------------------------------
+
```

Comment:
Can we make it clear that the configuration of the exporter is exclusive (we can sink traces to only one location) so that users don't wonder why, if they configure an OTEL collector sink, their traces aren't showing up in the MLflow UI?

Reply:
Makes sense!